src/crawlee/models.py

```diff
@@ -71,6 +71,13 @@ def from_url(
 
         return result
 
+    def get_query_param_from_url(self, param: str) -> str | None:
```

Comment:
Could we add a `default` parameter to this? It's against YAGNI, but I think it's still justifiable.

Reply:
hmm, what default would you add?

Reply:
I mean something like what `dict.get` goes. By default, the default should be `None`